Add BTT PITFT50 v2 Touch Display Mount

alanho opened the pull request:
<!--
Thank you for your interest in contributing to the VoronUsers repository, it is
highly appreciated!

**Please make sure the submission conforms to the rules outlined below. PRs which fail
to conform to the rules below are likely to be rejected.**
-->

  * [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

<!--
Describe the submission further using the template provided below. The more 
details the better!
-->

#### Which mods/configurations/profiles are added by this PR?
BTT PITFT50 v2 Touch Display Mount

#### How was it tested? 
Printed and Installed on my printer (as seen in the phoros)

#### Any background context you want to provide?
BTT PITFT50 v2 has a new layout, this design accommodates these changes

#### Screenshots (if appropriate)
in the README.md

#### Further notes

KiloQubit reviewed (changes requested):
Thanks for your mod!

Is `Voron-2.4-Mount-Generisch-v3.stl` modified from the `sttts` version, or is it the same?  If it's the same, then please remove it from your mod and instead put something in the README that directs users to print the mount from the `sttts` mod.  If it is different, it is not in a printable orientation.

In `BTT-PITFT50-v2-Mount.stl`, the undersides of the through holes have unprintable overhangs, they could be fixed by adding a 45 degree chamfer.

<img width="308" alt="Screen Shot 2021-06-08 at 19 46 39" src="https://user-images.githubusercontent.com/69389593/121271794-6fb75b80-c892-11eb-92ce-4c96b97ff970.png">

alanho commented:
Thanks for the feedback! I've added chamfer to the through holes, removed the Voron-2.4-Mount-Generisch-v3.stl and added instruction to redirect people to sttts's mod.

KiloQubit reviewed (approved):
Thanks for the update!

alanho commented:
just a heads up, due to my poor fusion 360 skills, i've exported a STL without opening for the thumbwheel and switch. i've creating another PR to fix this. sorry!
